### ISSUE TITLE
Vim-like key bindings 

### ISSUE THREAD

**Issue by Zebrazilla** [yakyak/yakyak#65]
Have anyone tried adding Vim-like key bindings to yakyak? I'd be very much interested in getting my hands on such a modification, even if it's barebones - this would up the effeciency a lot in conjunction with tiling window managers and a general keyboard-centric approach.

**Comment by davibe** [comment]
nobody tried :)

**Comment by modoj** [comment]
I love vim-mode ANYTHING. I just discovered this project, let me familiarize myself with it, if it's feasible and able to implement without breaking anything, and I'll think about it :)

I don't always use vim for everything, sometimes I want a GUI (I'm not fond of gVim), so I've been using Sublime_Text for the last couple month, and I love it. Especially, and really, the only reason I even consider it viable, VIM MODE! Using Firefox? Vimperator. And on Chromium I use Vimium. The way I see it, I already learned to control an interface with vim-like keybindings, why not use them everywhere text plays a front and center role?!

Sorry unrelated for the aside, especially since I'm brand-spanking new to the community, but if it's possible, I would love to make it happen. Again, let me look at the code first before I commit to such a thing. 

If there are specific features you would like, let me know what you are thinking.
